installation/cross_compile/README.md

```diff
@@ -0,0 +1,123 @@
+# Cross-Compilation
+Since the processing power of the Raspberry Pi Zero module is insufficient for "quick" compilation, we made tools to help you compile ROS2 on your PC for the Raspberry Pi Zero.
+Therefore, this tutorial will give you instructions on how to use the tools to cross-compile ROS2 for Raspberry Pi Zero (could be also extended to other Raspberry Pi versions).
+
+## Prerequisites
+Please make sure everything is ready for you to proceed:
+- Raspberry Pi OS ready on your Raspberry Pi ([tutorial](https://www.raspberrypi.org/documentation/installation/installing-images/)).
+- SSH access to Raspberry Pi ([tutorial](../README.md#wifi-and-ssh)).
+- Docker on your PC ([tutorial](https://docs.docker.com/get-docker/)).
+- Optional SSH server on your PC (`sudo apt install openssh-server`).
+
+## Raspberry Pi Preparation
+
+ROS2 requires the following packages to be installed on your Raspberry Pi:
+```bash
+sudo apt update
+
+# Compilation dependencies
+sudo apt install \
+    liblog4cxx-dev \
+    python3-dev
+
+# Runtime dependencies
+sudo apt install \
+    python3-numpy \
+    python3-netifaces \
+    python3-yaml
+
+# Optional tools
+sudo apt install sshfs
+```
+
+## ROS2 Cross-Compilation on Your PC
+
+Now we can start a process of ROS2 cross-compilation.
+The supplied Docker image is equipped with most of the tools you need, so start it with:
+```bash
+./start_docker.sh
```

Comment:
It `sudo` is required it means that the current user is not in `docker` user group. It usually means that Docker is not properly installed, maybe I should add this as a note

Reply:
Oh yes, you're right, it's not really our responsibility, but since we are going to deal with users not familiar with Docker it is probably worse adding a small note.